12LessonTwelve/in-class-project/index.html

```diff
@@ -7,8 +7,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link href="css/css-reset.css" rel="stylesheet">
-    <title></title>
-    <!-- Google font-->
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
     <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet">
     <link rel="stylesheet" href="css/style.css">
 </head>
```

Comment:
okie dokie, will do.
thanks
clint

On Tue, Sep 13, 2016 at 5:41 PM, Ethan Stenis <notifications@github.com>
wrote:

> In 12LessonTwelve/in-class-project/index.html
> <https://github.com/AustinCodingAcademy/intro-workbook/pull/86#discussion_r78658341>
> :
>
> > @@ -7,8 +7,7 @@
> >      <meta http-equiv="X-UA-Compatible" content="IE=edge">
> >      <meta name="viewport" content="width=device-width, initial-scale=1">
> >      <link href="css/css-reset.css" rel="stylesheet">
> > -    <title></title>
> > -    <!-- Google font-->
> > +    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
> >      <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet">
> >      <link rel="stylesheet" href="css/style.css">
> >  </head>
>
> [image: screen shot 2016-09-13 at 5 38 57 pm]
> <https://cloud.githubusercontent.com/assets/12698279/18493996/2e5536b0-79d9-11e6-8ea4-74ea6d3350bc.png>
>
> Please make adjustments to your pricing divs in both desktop and phone
> sizes. Once you can get the changes made, I can bump your grade up.
> Overall, though, good job.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/AustinCodingAcademy/intro-workbook/pull/86/files/621af19179d2e9adea2be286ff53ed6523a362c8..59b834cb1f28fa9d51bfb870afd7712d3ec29184#r78658341>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ATsR8k35CYCy8nIK4ahkUt-Z64Fzum5yks5qpyaNgaJpZM4JVvKV>
> .
>